### ISSUE TITLE
HTTP Auth pop-up when behind nginx with SSL

### ISSUE THREAD

**Issue by bubo77** [Graylog2/graylog2-server#2216]
### Problem description
Hi,

I've setup my graylog with SSL behind nginx.
But every time i reload the login page as an un authenticated user i get a pop HTTP auth window what i have to cancel
Note that our graylog setup with LDAP this pop up only accepts local graylog admin user doesn't accept LDAP user.

**Graylog config:**
```
rest_listen_uri = https://graylogmaster01.internal.test.xxx.io:12900/
#rest_transport_uri =
rest_enable_tls = true
rest_tls_cert_file = /etc/graylog/server/certs/graylogmaster01.internal.test.xxx.io.cert.pem
rest_tls_key_file = /etc/graylog/server/certs/graylogmaster01.internal.test.xxx.io.key.pkcs8.pem
web_listen_uri = http://127.0.0.1:9000/
```

**Nginx config:**
``` 
server {
   listen 443;
   server_name graylogmaster01.internal.test.xxx.io;
   ssl on;
       ssl_certificate /etc/ssl/graylog.test.xxx.io/graylog.test.xxx.io.pem;
       ssl_certificate_key /etc/ssl/graylog.test.xxx.io/graylog.test.xxx.io.key;
       ssl_session_cache   shared:SSL:5m;
       ssl_session_timeout 5m;
       ssl_prefer_server_ciphers On;
       ssl_protocols TLSv1 TLSv1.1 TLSv1.2;
       ssl_ciphers ECDH+AESGCM:DH+AESGCM:ECDH+AES256:DH+AES256:ECDH+AES128:DH+AES:ECDH+3DES:DH+3DES:RSA+AESGCM:RSA+AES:RSA+3DES:!aNULL:!MD5:!DSS;
   location / {
       proxy_pass http://127.0.0.1:9000/;
       proxy_set_header Host $host;
       proxy_set_header X-Real-IP $remote_addr;
       proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
       proxy_set_header X-Graylog-Server-URL https://graylogmaster01.internal.test.xxx.io/api;
       proxy_pass_request_headers on;
       proxy_connect_timeout 150;
       proxy_send_timeout 100;
       proxy_read_timeout 100;
       proxy_buffers 4 32k;
       client_max_body_size 8m;
       client_body_buffer_size 128k;
   }
   location /api/ {
       proxy_pass https://graylogmaster01.internal.test.xxx.io:12900/;
       proxy_set_header Host $host;
       proxy_set_header X-Real-IP $remote_addr;
       proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
       proxy_pass_request_headers on;
       proxy_connect_timeout 150;
       proxy_send_timeout 100;
       proxy_read_timeout 100;
       proxy_buffers 4 32k;
       client_max_body_size 8m;
       client_body_buffer_size 128k;
   }
}
```
**Error from chrome inspect:**
```
vendor.js:398Download the React DevTools for a better development experience: https://fb.me/react-devtools
plugin.org.graylog.plugins.pipelineprocessor.PipelineProcessorPlugin.62d4f9e….js:19484 GET https://graylogmaster01.internal.test.xxx.io/api/system/sessions 401 (Unauthorized)Request.end @ plugin.org.graylog.plugins.pipelineprocessor.PipelineProcessorPlugin.62d4f9e….js:19484(anonymous function) @ plugin.org.graylog.plugins.pipelineprocessor.PipelineProcessorPlugin.62d4f9e….js:13577tryCatcher @ plugin.org.graylog.plugins.pipelineprocessor.PipelineProcessorPlugin.62d4f9e….js:18208Promise._resolveFromResolver @ plugin.org.graylog.plugins.pipelineprocessor.PipelineProcessorPlugin.62d4f9e….js:16287Promise @ plugin.org.graylog.plugins.pipelineprocessor.PipelineProcessorPlugin.62d4f9e….js:15877Request.promise @ plugin.org.graylog.plugins.pipelineprocessor.PipelineProcessorPlugin.62d4f9e….js:13576Request.then @ plugin.org.graylog.plugins.pipelineprocessor.PipelineProcessorPlugin.62d4f9e….js:13611init @ plugin.org.graylog.plugins.pipelineprocessor.PipelineProcessorPlugin.62d4f9e….js:86079t @ vendor.js:415e.exports @ vendor.js:415(anonymous function) @ plugin.org.graylog.plugins.pipelineprocessor.PipelineProcessorPlugin.62d4f9e….js:86068__webpack_require__ @ plugin.org.graylog.plugins.pipelineprocessor.PipelineProcessorPlugin.62d4f9e….js:20Session @ plugin.org.graylog.plugins.pipelineprocessor.PipelineProcessorPlugin.62d4f9e….js:2322getStore @ plugin.org.graylog.plugins.pipelineprocessor.PipelineProcessorPlugin.62d4f9e….js:2382(anonymous function) @ plugin.org.graylog.plugins.pipelineprocessor.PipelineProcessorPlugin.62d4f9e….js:22273__webpack_require__ @ plugin.org.graylog.plugins.pipelineprocessor.PipelineProcessorPlugin.62d4f9e….js:20CurrentUser @ plugin.org.graylog.plugins.pipelineprocessor.PipelineProcessorPlugin.62d4f9e….js:2199getStore @ plugin.org.graylog.plugins.pipelineprocessor.PipelineProcessorPlugin.62d4f9e….js:2382(anonymous function) @ plugin.org.graylog.plugins.pipelineprocessor.PipelineProcessorPlugin.62d4f9e….js:1979__webpack_require__ @ plugin.org.graylog.plugins.pipelineprocessor.PipelineProcessorPlugin.62d4f9e….js:20(anonymous function) @ plugin.org.graylog.plugins.pipelineprocessor.PipelineProcessorPlugin.62d4f9e….js:1890__webpack_require__ @ plugin.org.graylog.plugins.pipelineprocessor.PipelineProcessorPlugin.62d4f9e….js:20(anonymous function) @ plugin.org.graylog.plugins.pipelineprocessor.PipelineProcessorPlugin.62d4f9e….js:1772__webpack_require__ @ plugin.org.graylog.plugins.pipelineprocessor.PipelineProcessorPlugin.62d4f9e….js:20(anonymous function) @ plugin.org.graylog.plugins.pipelineprocessor.PipelineProcessorPlugin.62d4f9e….js:1552__webpack_require__ @ plugin.org.graylog.plugins.pipelineprocessor.PipelineProcessorPlugin.62d4f9e….js:20(anonymous function) @ plugin.org.graylog.plugins.pipelineprocessor.PipelineProcessorPlugin.62d4f9e….js:345__webpack_require__ @ plugin.org.graylog.plugins.pipelineprocessor.PipelineProcessorPlugin.62d4f9e….js:20(anonymous function) @ plugin.org.graylog.plugins.pipelineprocessor.PipelineProcessorPlugin.62d4f9e….js:229__webpack_require__ @ plugin.org.graylog.plugins.pipelineprocessor.PipelineProcessorPlugin.62d4f9e….js:20(anonymous function) @ plugin.org.graylog.plugins.pipelineprocessor.PipelineProcessorPlugin.62d4f9e….js:57__webpack_require__ @ plugin.org.graylog.plugins.pipelineprocessor.PipelineProcessorPlugin.62d4f9e….js:20(anonymous function) @ plugin.org.graylog.plugins.pipelineprocessor.PipelineProcessorPlugin.62d4f9e….js:40(anonymous function) @ plugin.org.graylog.plugins.pipelineprocessor.PipelineProcessorPlugin.62d4f9e….js:43
plugin.org.graylog.plugins.pipelineprocessor.PipelineProcessorPlugin.62d4f9e….js:14572 Unhandled rejection SuperagentPromiseError: cannot GET https://graylogmaster01.internal.test.xxx.io/api/system/sessions (401)
    at https://graylogmaster01.internal.test.xxx.io/plugin/org.graylog.plugins.pip…pipelineprocessor.PipelineProcessorPlugin.62d4f9ed4c4bd0ccbd76.js:13580:20
    at Request.callback (https://graylogmaster01.internal.test.xxx.io/plugin/org.graylog.plugins.pip….pipelineprocessor.PipelineProcessorPlugin.62d4f9ed4c4bd0ccbd76.js:19328:4)
    at Request.<anonymous> (https://graylogmaster01.internal.test.xxx.io/plugin/org.graylog.plugins.pip…pipelineprocessor.PipelineProcessorPlugin.62d4f9ed4c4bd0ccbd76.js:19051:11)
    at Request.Emitter.emit (https://graylogmaster01.internal.test.xxx.io/plugin/org.graylog.plugins.pip…pipelineprocessor.PipelineProcessorPlugin.62d4f9ed4c4bd0ccbd76.js:19738:21)
    at XMLHttpRequest.xhr.onreadystatechange (https://graylogmaster01.internal.test.xxx.io/plugin/org.graylog.plugins.pip…pipelineprocessor.PipelineProcessorPlugin.62d4f9ed4c4bd0ccbd76.js:19410:11)
vendor.js:405 Warning: Failed propType: Invalid prop `href` of type `object` supplied to `Button`, expected `string`. Check the render method of `NodesActions`.t.production.t.env.NODE_ENV.r @ vendor.js:405s @ vendor.js:398c @ vendor.js:398v.createElement @ vendor.js:398NodesActions_render @ NodesActions.jsx:90y._renderValidatedComponentWithoutOwnerOrContext @ vendor.js:403y._renderValidatedComponent @ vendor.js:403ReactCompositeComponent__renderValidatedComponent @ vendor.js:12y.mountComponent @ vendor.js:403ReactCompositeComponent_mountComponent @ vendor.js:12a.mountComponent @ vendor.js:12v.Mixin.mountChildren @ vendor.js:404y.Mixin._createContentMarkup @ vendor.js:403y.Mixin.mountComponent @ vendor.js:403a.mountComponent @ vendor.js:12v.Mixin.mountChildren @ vendor.js:404y.Mixin._createContentMarkup @ vendor.js:403y.Mixin.mountComponent @ vendor.js:403a.mountComponent @ vendor.js:12y.mountComponent @ vendor.js:403ReactCompositeComponent_mountComponent @ vendor.js:12a.mountComponent @ vendor.js:12v.Mixin.mountChildren @ vendor.js:404y.Mixin._createContentMarkup @ vendor.js:403y.Mixin.mountComponent @ vendor.js:403a.mountComponent @ vendor.js:12y.mountComponent @ vendor.js:403ReactCompositeComponent_mountComponent @ vendor.js:12a.mountComponent @ vendor.js:12v.Mixin.mountChildren @ vendor.js:404y.Mixin._createContentMarkup @ vendor.js:403y.Mixin.mountComponent @ vendor.js:403a.mountComponent @ vendor.js:12y.mountComponent @ vendor.js:403ReactCompositeComponent_mountComponent @ vendor.js:12a.mountComponent @ vendor.js:12y.mountComponent @ vendor.js:403ReactCompositeComponent_mountComponent @ vendor.js:12a.mountComponent @ vendor.js:12v.Mixin.mountChildren @ vendor.js:404y.Mixin._createContentMarkup @ vendor.js:403y.Mixin.mountComponent @ vendor.js:403a.mountComponent @ vendor.js:12y.mountComponent @ vendor.js:403ReactCompositeComponent_mountComponent @ vendor.js:12a.mountComponent @ vendor.js:12v.Mixin.mountChildren @ vendor.js:404y.Mixin._createContentMarkup @ vendor.js:403y.Mixin.mountComponent @ vendor.js:403a.mountComponent @ vendor.js:12y.mountComponent @ vendor.js:403ReactCompositeComponent_mountComponent @ vendor.js:12a.mountComponent @ vendor.js:12v.Mixin.mountChildren @ vendor.js:404y.Mixin._createContentMarkup @ vendor.js:403y.Mixin.mountComponent @ vendor.js:403a.mountComponent @ vendor.js:12y.mountComponent @ vendor.js:403ReactCompositeComponent_mountComponent @ vendor.js:12a.mountComponent @ vendor.js:12y._updateRenderedComponent @ vendor.js:403y._performComponentUpdate @ vendor.js:403y.updateComponent @ vendor.js:403ReactCompositeComponent_updateComponent @ vendor.js:12y.performUpdateIfNecessary @ vendor.js:403a.performUpdateIfNecessary @ vendor.js:13s @ vendor.js:12r.perform @ vendor.js:13r.perform @ vendor.js:13h.perform @ vendor.js:12L @ vendor.js:12ReactUpdates_flushBatchedUpdates @ vendor.js:12r.closeAll @ vendor.js:13r.perform @ vendor.js:13p.batchedUpdates @ vendor.js:398c @ vendor.js:12o @ vendor.js:14p.enqueueSetState @ vendor.js:14o.setState @ vendor.js:398n @ vendor.js:415r.emit @ vendor.js:409e.exports.trigger @ vendor.js:415(anonymous function) @ plugin.org.graylog.plugins.pipelineprocessor.PipelineProcessorPlugin.62d4f9e….js:21992tryCatcher @ plugin.org.graylog.plugins.pipelineprocessor.PipelineProcessorPlugin.62d4f9e….js:18208Promise._settlePromiseFromHandler @ plugin.org.graylog.plugins.pipelineprocessor.PipelineProcessorPlugin.62d4f9e….js:16314Promise._settlePromiseAt @ plugin.org.graylog.plugins.pipelineprocessor.PipelineProcessorPlugin.62d4f9e….js:16388Promise._settlePromises @ plugin.org.graylog.plugins.pipelineprocessor.PipelineProcessorPlugin.62d4f9e….js:16504Async._drainQueue @ plugin.org.graylog.plugins.pipelineprocessor.PipelineProcessorPlugin.62d4f9e….js:13794Async._drainQueues @ plugin.org.graylog.plugins.pipelineprocessor.PipelineProcessorPlugin.62d4f9e….js:13804Async.drainQueues @ plugin.org.graylog.plugins.pipelineprocessor.PipelineProcessorPlugin.62d4f9e….js:13686

```
### Steps to reproduce the problem
 
1. open the page https://graylog
2. Make sure you are logged out of Graylog / empty browser cache
3. Refresh the page
4. Basic HTTP Auth pop up overlays the graylog login frame
5. Cancel it you get the above error message
6. When you login with LDAP user everything works

### Environment

* Graylog Version: 2.0.0
* Elasticsearch Version: 2.3.2
* MongoDB Version: 2.6.11
* Operating System: Centos 7 + SELinux
* Browser version: Chrome 50

**Comment by bubo77** [comment]
Upgraded to 2.0.1 it resolved the pop-up Basic HTTP auth

**Comment by joschi** [comment]
@bubo77 Thanks for the update! Since the problem has been solved by upgrading to Graylog 2.0.1, I'll close this issue.
